src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/NewDiscountProgramModerator.cs

```diff
@@ -6,7 +6,7 @@ namespace Exadel.CrazyPrice.Common
     /// <summary>
     /// Gets/sets information for a Moderator
     /// </summary>
-    public class NewDiscountProgram
+    public class NewDiscountProgramModerator
```

Comment:
Probably rename to NewDiscountProgramRequest

Reply:
[It fixed](https://github.com/yegorl/exadel_discounts_be/blob/fix/src/Exadel.CrazyPrice/Exadel.CrazyPrice.Common/NewDiscountProgramRequest.cs)